meeshkan/serve/mock/specs.py

```diff
@@ -16,27 +18,51 @@ class OpenAPISpecification:
     source: str
 
 
-def load_specs(specs_dir: str) -> Sequence[OpenAPISpecification]:
-    if not os.path.exists(specs_dir):
-        logging.info("OpenAPI schema directory not found %s", specs_dir)
-        return []
+def load_specs(specs: Union[str, Sequence[str]]) -> Sequence[OpenAPISpecification]:
+    if isinstance(specs, str):
+        specs = (specs,)
+
     specs_with_sources = []
-    specs_paths = [
-        s
-        for s in os.listdir(specs_dir)
-        if s.endswith("yml") or s.endswith("yaml") or s.endswith("json")
-    ]
-    for spec_path in specs_paths:
-        full_spec_path = os.path.join(specs_dir, spec_path)
-        with open(full_spec_path, encoding="utf8") as spec_file:
+
+    for spec in specs:
+        is_http = spec.startswith("http://") or spec.startswith("https://")
```

Comment:
Could all of this be put under a separate `load_spec` function and maybe split into smaller helpers? it's maybe a bit long to live under a for loop.

Reply:
Done!